Frontend/src/Components/Temperature/TempView.js

```diff
@@ -0,0 +1,70 @@
+import {Flex, VStack, Box} from "@chakra-ui/react";
+import RangeRow from "../Shared/RangeRow";
+import CONSTANTS from "../../data-constants.json";
+
+
+export default function TempView(props) {
+
+    console.log(props.data);
+    console.log(CONSTANTS)
+    return (
+        <Flex flex="auto" direction="column" overflowY="scroll">
+            <RangeRow
+                dataConstant={CONSTANTS.motor_temp}
+                dataValue={props.data.motor_temp[0] ?? CONSTANTS.motor_temp.MIN}
+                dataTitle="Motor"
+                size="sm"
+            />
+            <RangeRow
+                dataConstant={CONSTANTS.road_temp}
+                dataValue={props.data.road_temp[0] ?? CONSTANTS.road_temp.MIN}
+                dataTitle="Road"
+                size="sm"
+            />
```

Comment:
```suggestion
            <RangeRow
                dataConstant={CONSTANTS.road_temp}
                dataValue={props.data.road_temp[0] ?? CONSTANTS.road_temp.MIN}
                dataTitle="Road"
                size="sm"
                bg={headerBg}
            />
```
Another suggestion for the reimplementation. We should alternate background colors for every other row. `headerBg` will have to be set using `getColorMode()` and `getColor()` (see [`BatteryPack.js`](https://github.com/badgerloop-software/chase-car-dashboard/blob/main/Frontend/src/Components/BatteryCells/BatteryPack.js))